gnocchi/tests/base.py

```diff
@@ -234,14 +264,6 @@ class TestCase(base.BaseTestCase):
     @classmethod
     def setUpClass(self):
         super(TestCase, self).setUpClass()
-
-        # NOTE(sileht): oslotest does this in setUp() but we
-        # need it here
```

Comment:
I haven't test but I think all the logs/print done during service.prepare_service and indexer setup will be printed.